#! /bin/sh /usr/share/dpatch/dpatch-run
## upstream-fix-typos by Sven Mueller <sven@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Upstream fix for some typos in manpages for reconstruct and chk_cyrus

@DPATCH@
--- a/man/chk_cyrus.8
+++ b/man/chk_cyrus.8
@@ -40,7 +40,7 @@
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
 .SH NAME
-chk_cyrus \- perform a consistancy check of the cyrus mailstore
+chk_cyrus \- perform a consistency check of the cyrus mailstore
 .SH SYNOPSIS
 .B chk_cyrus
 [
@@ -57,7 +57,7 @@ chk_cyrus \- perform a consistancy check
 ]
 .SH DESCRIPTION
 .I Chk_cyrus
-is used to perform a consistancy check on the cyrus datastore, and output
+is used to perform a consistency check on the cyrus datastore, and output
 a list of files/directories that are expected to exist, but do not.
 Status messagess are output to stderr, the list of files/directories is
 output to stdout.  This list can be passed to a backup program to aid a
@@ -73,10 +73,10 @@ file unless specified otherwise by \fB-C
 Read configuration options from \fIconfig-file\fR.
 .TP
 .BI \-P " partition"
-Run the consistancy check for only the given partition.
+Run the consistency check for only the given partition.
 May not be specified with -M.
 .BI \-M " mailbox"
-Run the consistancy check for only the given mailbox.
+Run the consistency check for only the given mailbox.
 May not be specified with -P.
 .SH FILES
 .TP
--- a/man/reconstruct.8
+++ b/man/reconstruct.8
@@ -39,7 +39,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.\" $Id: reconstruct.8,v 1.19 2004/10/13 18:07:49 shadow Exp $
+.\" $Id: reconstruct.8,v 1.20 2006/06/19 13:01:19 murch Exp $
 .SH NAME
 reconstruct \- reconstruct mailboxes
 .SH SYNOPSIS
@@ -118,8 +118,8 @@ Create the mailboxes in the database in
 .B \-x
 When processing a mailbox which is not in the mailbox list (e.g. via
 the -p or -f options), do not import the metadata from the mailbox,
-instead create it anew (this specifically affects atleast the mailbox's seen
-stat unique identifier, user flags, and ACL).
+instead create it anew (this specifically affects at least the mailbox's seen
+state unique identifier, user flags, and ACL).
 .TP
 .B \-r
 Recursively reconstruct all sub-mailboxes of the mailboxes or mailbox
